skin-schema.json

```diff
@@ -716,6 +716,36 @@
         }
       }
     },
+    "isVrAnimationEnabled": {
+      "description": "Shows if vr inform animation is enabled",
+      "type": "object",
+      "properties": {
+        "vrNotification": {
+          "description": "Vr notification should be shown",
+          "type": "boolean",
+          "default": "If no value is provided for vrNotification, vrNotification will default to true."
+        },
+        "vrIcon": {
+          "description": "Vr icon should be  shown",
+          "type": "boolean",
+          "default": "If no value is provided for vrIcon, vrIcon will default to false."
+        }
+      }
+    },
+    "animationDurations": {
```

Comment:
This should be `animationDuration`

Reply:
variable name is animationDurationS :-) because it is object with several durations.

Reply:
Oh ok, I see. Looks good then.